client/components/app.jsx

```diff
@@ -46,13 +48,31 @@ class App extends React.Component {
       });
   }
 
+  deleteAGrade(userId) {
+    const newGrades = this.state.grades;
+    fetch(`/api/grades/${userId}`, {
+      method: 'DELETE'
+    })
+      .then(res => res.json())
+      .then(() => {
+        for (let i = 0; i < newGrades.length; i++) {
+          if (newGrades[i].id === userId) {
+            newGrades.splice(i);
+          }
+        }
```

Comment:
Be sure to make a copy of the array first, so you're not directly mutating state.